### DISCUSSION TITLE
support request for Nix language server `nixd`

### DISCUSSION THREAD

**Discussion by peterbecich** [joaotavora/eglot#1284] [Ideas]
I see Eglot currently supports `rnix`: https://github.com/nix-community/rnix-lsp

Can it also support `nixd`? https://github.com/nix-community/nixd

Thank you

**Comment by joaotavora** [comment]
Hello.  It possibly/probably already supports it, but if you mean "out of the box support" where you don't have to write a single line adding `nixd` to `eglot-server-programs`, please send mail to bug-gnu-emacs@gnu.org requesting this, along with a patch for `lisp/progmodes/eglot.el` where you do that addition to the default value of `eglot-server-programs`.  You can `CC` me in the email.
